newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/index.js

```diff
@@ -421,7 +421,7 @@ const BuildSection = ({
   const shouldDisplayAnnouncements =
     !authenticatedUser.limits ||
     !authenticatedUser.limits.capabilities.classrooms ||
-    !authenticatedUser.limits.capabilities.classrooms.hideCommunityTab;
+    !authenticatedUser.limits.capabilities.classrooms.hidePlayTab;
```

Comment:
we probably should create a proper `announcement` capability to avoid relying on tabs which an disappear